loader/readsource.go

```diff
@@ -16,6 +16,7 @@ import (
 func ReadSource(
 	logger logrus.FieldLogger, src, pwd string, filesystems map[string]fsext.Fs, stdin io.Reader,
 ) (*SourceData, error) {
+	pwdURL := &url.URL{Scheme: "file", Path: filepath.ToSlash(filepath.Clean(pwd)) + "/"}
```

Comment:
```suggestion
	pwdURL := &url.URL{Scheme: "file", Path: filepath.Clean(pwd) + "/"}
```
If I remove `ToSlash` I don't get any test failing. Why is it required? Isn't Clean enough?

Reply:
ToSlash is about windows support. To be honest it might not be required, but as this is just a copy from a few lines below and the path related issues with windows are a nightmare tofix ... I prefer not to touch it :sweat_smile:

Reply:
Aaah ... actually it is required because on windows Clean returns paths with `\` but that isn't a separator in URLs

Reply:
Can you add a comment on it, please? So, the next time I read it I have at least an idea of what to research for. :pray:

Reply:
Fine to me